vp/vp/settings.py

```diff
@@ -37,7 +37,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-
+    'rest_framework',
```

Comment:
I think this is a holdover from your last PR and is now extraneous?

Reply:
Yep, that is correct. I just removed it.